flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/ForStOptions.java

```diff
@@ -173,7 +183,7 @@ public class ForStOptions {
                                     "This determines the factory for timer service state implementation.");
 
     /** The cache size per key-group for ROCKSDB timer service factory implementation. */
-    @Documentation.Section(Documentation.Sections.STATE_BACKEND_ROCKSDB)
```

Comment:
are we losing the rocksdb information here?

Reply:
Nope. Change the section that contains this doc entry, from RocksDB to ForSt.